$OpenBSD: patch-hpodder_cabal,v 1.7 2015/09/25 20:04:03 kili Exp $

Unbreak build with HaXml>=1.22

Pull in network-uri (which had bben split off the network package)

Turn on FlexibleContexts to let it build with ghc-7.10

--- hpodder.cabal.orig	Tue Nov  8 16:49:38 2011
+++ hpodder.cabal	Thu Sep 24 22:35:48 2015
@@ -68,8 +68,8 @@ Description:  Podcasting is a method of publishing rad
  hpodder is SAFE and is designed with data integrity in mind from the
  beginning.  It should be exceedingly difficult to lose a podcast
  episode, even in the event of a power failure.
-Build-Depends: network, unix, parsec, MissingH>=1.0.0,
- HDBC>=1.1.0, HDBC-sqlite3>=1.1.0, mtl, base>=3 && < 5, HaXml >= 1.20 && < 1.21, hslogger,
+Build-Depends: network, network-uri, unix, parsec, MissingH>=1.0.0,
+ HDBC>=1.1.0, HDBC-sqlite3>=1.1.0, mtl, base>=3 && < 5, HaXml >= 1.22, hslogger,
  ConfigFile, filepath, old-time, directory, process
 
 Executable: hpodder
@@ -81,4 +81,4 @@ Other-Modules: Commands, Config, DB, Download, Downloa
         Commands.Setup, Commands.Update
 GHC-Options: -O2
 Extensions: ExistentialQuantification, OverlappingInstances,
-    UndecidableInstances
+    UndecidableInstances, FlexibleContexts
